src/taco-remote-lib/ios/ios.ts

```diff
@@ -230,14 +229,18 @@ class IOSAgent implements ITargetPlatform {
     }
 
     public debugBuild(buildInfo: utils.BuildInfo, req: Express.Request, res: Express.Response): void {
-        if (this.webProxyInstance) {
-            this.webProxyInstance.kill();
-            this.webProxyInstance = null;
+        if (!global.tacoRemoteLib) {
+            global.tacoRemoteLib = {};
+        }
+
+        if (global.tacoRemoteLib.webProxyInstance) {
+            global.tacoRemoteLib.webProxyInstance.kill();
+            global.tacoRemoteLib.webProxyInstance = null;
         }
 
         var portRange: string = util.format("null:%d,:%d-%d", this.webDebugProxyDevicePort, this.webDebugProxyPortMin, this.webDebugProxyPortMax);
         try {
-            this.webProxyInstance = child_process.spawn("ios_webkit_debug_proxy", ["-c", portRange]);
+            global.tacoRemoteLib.webProxyInstance = child_process.spawn("ios_webkit_debug_proxy", ["-c", portRange]);
```

Comment:
If it lacks an `on("error"` handler then it will throw an exception, but we have a try/catch here which I believe should handle it. I think it's something worth doing, but I'm not sure whether it is worth mixing in with the ask-mode fix?